Source/Urho3D/Resource/Image.cpp

```diff
@@ -704,6 +725,16 @@ bool Image::BeginLoad(Deserializer& source)
             components_ = 4;
             break;
 
+        case 22:
+            compressedFormat_ = CF_ETC2_RGB;
+            components_ = 3;
+            break;
+
+        case 23:
+            compressedFormat_ = CF_ETC2_RGBA;
+            components_ = 4;
+            break;
+
```

Comment:
I actually wanted to ask why there is duplication here but forgot to ask it in the earlier review. Care to enlighten us? Perhaps you might as well explain the reason as code comment, if they are intentional.

Reply:
@weitjong  Both .ktx files and .pvr files can support etc2 image format, their difference is just the header.

Reply:
I see. If you could add a short comment line to say which is which that would be great.